### PR TITLE
docs: reference complementary plugins

### PR DESCRIPTION
**Pull request by mcarvin8**
# Explain your changes

---

Adds complementary plugins to the SGD README.

Per discussion in https://github.com/scolladon/sfdx-git-delta/issues/913

# Does this close any currently open issues?

---

Documentation only update.

# Any particular element that can be tested locally

---

# Any other comments

---

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -619,3 +620,10 @@ See [contributing.md](CONTRIBUTING.md) for sgd contribution principles.
 ## License
 
 This project license is MIT - see the [LICENSE.md](LICENSE.md) file for details
+
+## Complimentary Plugins
+
+These plugins have been designed to work with SGD:
+
+- [apex-test-list](https://github.com/renatoliveira/apex-test-list) - Developer: [renatoliveira](https://github.com/renatoliveira) - This plugin determines the specified Apex tests by reading test annotations made anywhere inside your Apex classes. You can have this plugin scan the package.xml created by SGD to determine the required Apex tests to run during deployment.
+- [apex-tests-git-delta](https://github.com/mcarvin8/apex-tests-git-delta) - Developer: [mcarvin8](https://github.com/mcarvin8) - This plugin determines the specified Apex tests by reading the commit messages in the commit range. You can use the same `--from` and `--to` commit hashes when using SGD and apex-tests-git-delta to determine the required Apex tests to run during deployment.
```
Comment (scolladon) [inline README.md:629]: **suggestions:** could it be moved just before the `Changelog` section please ?

File: README.md [hunk 2]
```diff
@@ -56,6 +56,7 @@
 - [Authors](#authors)
 - [Contributing](#contributing)
 - [License](#license)
+- [Complimentary Plugins](#complimentary-plugins)
```
Comment (scolladon) [inline README.md:59]: *praises:** nice, you did not forget this, thanks

**Review by scolladon — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -619,3 +620,10 @@ See [contributing.md](CONTRIBUTING.md) for sgd contribution principles.
 ## License
 
 This project license is MIT - see the [LICENSE.md](LICENSE.md) file for details
+
+## Complimentary Plugins
```
Comment (scolladon) [inline README.md:624]: **typo:** compl**e**mentary instead of compl**i**mentary ?

Comment (mcarvin8) [reply README.md]: fixed. thanks, yikes i missed that

File: README.md (hunk 1 above)
Comment (mcarvin8) [reply README.md]: moved before changelog

**Review by scolladon — APPROVED** [review]
LGTM !

Thanks for this contribution

**Comment by mcarvin8** [comment]
thanks, just amended the commit message to match what your changed MR title to

**Pull request merged**